src/navigator.js

```diff
@@ -105,18 +147,37 @@ module.exports = function(server, path, runtime) {
 /**
  * Renders the node listing page.
  * @param network the active network containing the node
+ * @param facades hash of RDF pipline component facades
  * @param index page template in hbs
  * @param path the prefix of the URL before the nodeId
  * @param qs query string from the request (including the ?)
  * @param res response object to write to
  */
-function listing(network, index, path, qs, res) {
+function listing(network, facades, index, path, qs, res) {
     res.setHeader('Content-Type', 'text/html');
+
+    // Generate a version of the nodes with the profiler summary metrics
+    var nodes = _.clone(network.graph.nodes);
+    network.graph.nodes.forEach(function(node) { 
+        if (! ( _.isEmpty(facades) || _.isUndefined(facades[node.id]))) { 
+            var metrics = facades[node.id].profiler.metrics;
+            _.extend( node,
```

Comment:
It does not modify the existing object.  On line 160, you can see I'm cloning it.  That is making a shallow copy and then making modifications to the copy.